doc/roadmap.rst

```diff
@@ -0,0 +1,80 @@
+Arviz Roadmap
+=============
+
+This is super preliminary and reflects thoughts on what needs to be done
+to get to a stable release as of August 1, 2018.
+
+Road to v1.0
+------------
+
+I would guess that this all takes ~6ish months to do. The initial PyPI
```

Comment:
six months from now seems like a lot, I will have more time to contribute starting next week. What about two months ( around the first week of October)?

Reply:
6 months was including getting releases of pymc3 and pystan out. i'm cautiously optimistic we can have something on pypi in a month, and something good on pypi in 2 months, though!

Reply:
Nice!